### DISCUSSION TITLE
Welcome to microservice-boilerplate Discussions!

### DISCUSSION THREAD

**Discussion by shsma** [shsma/microservice-boilerplate#5] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by filipe1992** [comment]
Your wok really help me to do a MicroService, and I want to understand more.
I want to know more about the use of Redis, with laravel.
Thank you for this.
Will you finish this work?
